components/ftx/common/BUILD.gn

```diff
@@ -0,0 +1,9 @@
+import("//brave/components/ftx/browser/buildflags/buildflags.gni")
+
+assert(ftx_enabled)
+
+source_set("common") {
```

Comment:
since these are only used in `browser` this component doesn't need browser and common
"A component used only by the browser process contains code directly in its directory." - https://www.chromium.org/developers/design-documents/cookbook